### ISSUE TITLE
Audio track from an external file.

### ISSUE THREAD

**Issue by Alexander-Kane** [kaltura/nginx-vod-module#1545]
Hello.

I have 1 video file and 2 audio files in different languages:
lesson_5.mp4 - only video, no audio.
lesson_5_eng.aac - audio track in English.
lesson_5_deu.aac - audio track in German.

How do I output the video and audio file together in one manifest?
The documentation says you can: "Muxing together audio and video streams from separate files".
I've read the documentation several times, but I can't figure it out.

https://site.com/hls/lessons/lesson_5.mp4/master.m3u8 - video works fine, but no sound.
How can I do add the audio from the lesson_5_eng.aac file?

**Comment by Alexander-Kane** [comment]
I tried it this way https://site.com/hls/lessons/,lesson_5.mp4,lang/eng/lesson_5_eng.aac,.urlset/master.m3u8 but I'm getting an error 404.
Tried to pack the .aac into a .mp4 container, then the link worked:
https://site.com/hls/lessons/,lesson_5.mp4,lang/eng/lesson_5_eng.mp4,.urlset/master.m3u8

So .aac audio is not supported? Need only .mp4? That's sad.

**Comment by erankor** [comment]
Yes, the only supported container formats are MP4 and MKV (experimental).
